ui_test/POM_hiway_asg/test_scripts/test_hiway_selenium.py

```diff
@@ -0,0 +1,203 @@
+import logging
+import pytest
+import csv
+import time
+from POM_hiway_asg.pages.login_using_google import LoginWithGoogle as LWG
+from POM_hiway_asg.pages.timesheet import TimesheetOptions
+from POM_hiway_asg.pages.constant_var import *
+
+logging.basicConfig(filename=HIWAY_LOG_FILE_NAME,
+                    format=LOGGER_LOG_ENTRY_FORMAT,
+                    datefmt=LOGGER_DATE_ENTRY_FORMAT)
+
+
+class TestHiway():
+    logger = logging.getLogger()
+    logger.setLevel(logging.ERROR)
+
+    @pytest.fixture(autouse=True)
+    def setup(self, setUp):
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this parameter "setUp" to match the regular expression ^[_a-z][a-z0-9_]*$. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=python%3AS117)